library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aldebaran/110CU/Equality/aldebaran_Cijk_Ailk_Bljk_HHS_BH_Bias_AH_SAV.yaml

```diff
@@ -3305,9 +3305,8 @@
     EdgeType: ShiftPtr
     EnableMatrixInstruction: true
     ExpandPointerSwap: 0
-    GlobalReadVectorWidthA: 8
-    GlobalReadVectorWidthA: 4
     GlobalReadPerMfma: 1
+    GlobalReadVectorWidthA: 4
```

Comment:
Removed the duplicate `GlobalReadVectorWidthA` key to ensure the correct value of `4` is used, as the latter key overrides the former in the YAML configuration.